src/components/LateralBar/LateralBar.tsx

```diff
@@ -1,15 +1,28 @@
-import React, {useState} from 'react';
+import React, { useState, useEffect } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faBars, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
 import './LateralBar.scss';
-
+import { ApiService } from '../../services/ApiService';
+ 
 interface Props { onClick: any, onSelect: any }
 
-
-
-
 export const LateralBar = ({ onClick, onSelect }: Props): JSX.Element => {
-    const [device, setDevice] = useState('');
+    const [device, setDevice] = useState(ApiService.user);
+    const [deviceList, setDeviceList] = useState([]);
+
+    const handleOnClick = (target: any) => {
+        onSelect(target.value);
+        setDevice(target.value);
+        onClick()
+    };
+
+   
+    useEffect((): any => {
+        ApiService.setDevice(device);
+        ApiService.getUsersList().then((apiResponse: any) => {
+            setDeviceList(apiResponse.map((value: string) => <li><button  className={`border-0 my-2 ${device === value && 'menu-button-actived'} menu-button }`} p-0 value={value} onClick={({ target }: any) => handleOnClick(target)}>{value}</button></li>))
```

Comment:
this line is toooooo long :D I would put the `<li>...` into another function and call it here